ajax/libs/cssobj/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "cssobj",
+  "description": "Runtime CSS manager, Turn CSS into dynamic JS module, Stylesheet CRUD (Create, Read, Update, Delete) in CSSOM, Solve common problems of CSS-in-JS, name space (local) class names",
+  "filename": "cssobj.min.js",
+  "keywords": [
+    "cssinjs",
+    "css-in-js",
+    "css in js",
+    "cssom",
+    "auto-prefixer",
+    "css generate",
+    "diff css",
+    "css modules"
+  ],
+  "author": "James Yang <jamesyang999@gmail.com>",
+  "homepage": "https://cssobj.github.io/cssobj-demo/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cssobj/cssobj.git"
+  },
+  "npmName": "cssobj",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/cssobj.!(amd|cjs|es)*"
```

Comment:
@sashberd @PeterDaveHello In fact, it's all the standard format generated from [rollup](https://github.com/rollup/rollup/wiki/JavaScript-API#format):

> iife – A self-executing function, suitable for inclusion as a <script> tag. (If you want to create a bundle for your application, you probably want to use this, because it leads to smaller file sizes.)
> umd – Universal Module Definition, works as amd, cjs and iife all in one

`iife` similar to `umd`, but smaller in size, without [overhead](https://github.com/cssobj/cssobj/blob/master/dist/cssobj.umd.js#L18-L22) when include in `<script>` tag directly, so the `min.js` generated from `iife` to get minimal file size. That's all the relation. In fact, you can use `umd` instead of `amd` too, but why rollup introduced `umd`? there's a reason that `umd` has some overhead when people want to optimize file size. If you want a minimal link, `umd+min` is enough for directly include in browsers. I like `iife`, it's simplified, so I generated in there, and include `iife` in **all** places when I want to demonstrate this lib...